encoders/image/CustomKerasImageEncoder/tests/test_customtfkeras.py

```diff
@@ -0,0 +1,79 @@
+import tempfile
+import os
+
+import numpy as np
+
+from .. import CustomKerasImageEncoder
+from jina.executors.metas import get_default_metas
+from tests.unit.executors.encoders.image import ImageTestCase
+from jina.executors import BaseExecutor
+
+
+def rm_files(file_paths):
+    for file_path in file_paths:
+        if os.path.exists(file_path):
+            if os.path.isfile(file_path):
+                os.remove(file_path)
+            elif os.path.isdir(file_path):
+                shutil.rmtree(file_path, ignore_errors=False, onerror=None)
+
+class TestNet:
+    def __init__(self):
+        self.model = None
+        self.input_shape = (224, 224, 3)
+        self.conv = Conv2D(32, (3, 3), padding='same', name='conv1', input_shape=self.input_shape)
+        self.activation_relu = Activation('relu')
+        self.flatten = Flatten()
+        self.dense = Dense(10, name='dense')
+        self.activation_softmax = Activation('softmax')
```

Comment:
missing imports

Reply:
fixed imports, added to the start of the file instead of get_encoder